src/frontend/komponenter/Brevmottakere/VergerOgFullmektigeFraRegister.tsx

```diff
@@ -82,6 +85,9 @@ export const VergerOgFullmektigeFraRegister: FC<Props> = ({
             ) : (
                 <BodyShort>Ingen verge/fullmektig i register</BodyShort>
             )}
+            {hentFullmektigeFeil && (
+                <ErrorMessage>Henting av fullmakter feilet: {hentFullmektigeFeil}</ErrorMessage>
+            )}
```

Comment:
`<Feilmelding/>` komponenten er egentlig laget for å slippe å ha sjekker på om feil finnes alle steder. Den ville ikke fungert her pga. defaultteksten før feilmeldingen, men vi kunne vurdert om denne skulle hatt muligheten for en prefix slik at man kan sende inn default-teksten her.

Reply:
Tbh syntes jeg det er mer lesbart å bare bruke den konstruksjonen slik som den er nå. Dette er en veldig standard måte å deale med feil på.